Make Steering Committee guidelines relevant (#44)
ansible/community-docs: steering_committee_membership_guidelines.rst

#. The initiator reports the case to the Committee via email.

#. The Committee discusses the case internally, evaluates its severity, and possible solutions.

#. If the committee concludes that the violation is not severe, it develops a proposal to the person on how the situation can be corrected and further interactions with the Community improved.

#. A Committee representative reaches out to the person with the proposal.

#. The removal process starts if:

  * The Committee decided that the severity of the violation excludes a possibility of further membership.

  * The person does not respond to the proposal.

  * The person explicitly rejects the proposal.

In case of starting the removal process, the topic's description in the reason's part changes correspondingly.

Chairperson
===========

Tadej Borovšak is elected as the chairperson for the year 2021-2022.

The chairperson election will happen once a year around the time of
Ansible Fest. In case when the current chairperson stepped down, the election happens immediately.

The process of the election consist of the following steps:

> #. Interested members who would not mind to be a chairperson will inform a

Reviewer: ```suggestion
#. Members interested in being the chairperson will inform a
```

3. Asynchronously participate in discussing and voting on the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_.
4. Review other proposals which importance needs the Committee's attention and provide feedback.
5. Act for the sake of the Community not promoting corporate or individual agenda during the decision making process.
6. Engage with the Community in a professional and positive manner, encourage community members to express their opinion.

.. _Joining the committee:

Joining the Steering Committee:
===============================

Eligibility:
~~~~~~~~~~~~

A person is eligible to become a Committee member if they have:

1. A wide knowledge of Ansible and / or its related projects.
2. Active contributions to the Ansible and / or related projects in any form described in the `Ansible Contributing Guidelines <https://github.com/ansible/community-docs/blob/main/contribution_to_project.rst>`_.
3. A consent to follow the :ref:`Expectations for the Committee members<Expectations for members>`.

Process:
~~~~~~~~

1. Any community member may nominate someone/themselves for committee

>    membership at any time by contacting one or more of the current members (see the list `here <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst#members>`_) or by sending an email to ``ansible-community@redhat.com``.

Author: Much better now:) Thanks, committed.

>    membership at any time by contacting one or more of the current members (see the list `here <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst#members>`_) or by sending an email to ``ansible-community@redhat.com``.

Reviewer 2: Actually we should **never** use `here` as a link title :) (see also https://github.com/ansible/ansible/issues/73510)

>    membership at any time by contacting one or more of the current members (see the list `here <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst#members>`_) or by sending an email to ``ansible-community@redhat.com``.

Reviewer 2: @gundalow sorry, somewhat misread you, since you are missing a `not` :)

>    membership at any time by contacting one or more of the current members (see the list `here <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst#members>`_) or by sending an email to ``ansible-community@redhat.com``.

Author: heh:)

rejoin the Committee by following the :ref:`process for joining the Steering Committee<Joining the committee>`.

In case of resignation, the Committee member should inform other members by email that they plan to step down.

Involuntarily Leaving the Steering Committee:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

A Committee member will be removed from the Committee if they:

1. Do not participate in asynchronous discussing and voting on the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_ for more than 3 months in a row.
2. Participate unreasonably irregularly (for example, once a month for several months). Unreasonability is defined by other Committee members considering circumstances in each particular case.
3. Violate the `Ansible Community Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_.

The removal process (in all cases except resignation) consists of the
following steps:

1. Another Committee member initiates the process by starting a dedicated email thread. It must be sent to all the current Committee members including the person that can be removed.

> 2. The Committee members discuss the removal and vote in the email thread.

Reviewer 2: The old version made this a very open and transparent process (by creating an issue in community-topics and having the discussion and vote there). While I prefer to have at least the initial discussion not happening that openly - that can quickly get very damaging by rash comments -, I think it is good that the final discussion and vote should be done openly, or at least its results be published (rationale why the member was excluded, i.e. what exactly was voted on, and including voting result).

Author: This is a really good question how we should handle it.
When I was changing this i was thinking about possible hurt that it can inflict to excluded people as it can look a bit like public condemnation.
Of course, we would try to use proper wording but anyway "the person has been excluded from the Committee because they have regularly missed the topics" doesn't sound good:) 
Transparency is great but i'd suggest avoiding possible hurt if possible..
Possibility of public exclusion can be also an obstacle for potential members to come if they are not 100% sure they can take part in discussing every topic.
I'm not strictly against of keeping it public but i'd suggest taking the points above into consideration.

@felixfontein @gundalow @briantist @tadeboro any thoughts?

Author: Or, for example, a topic with a title `Exclusion of Andersson007 from the Steering Committee`:) doesn't bring good feelings:)

Reviewer 3: This is a sensitive topic, it's good to bring it up. 

On the one hand, I do think transparency is important: giving the impression that there are secret discussions that can result in a member being removed also does not bring good feelings.

But on the other, Andrew has a good point that an open discussion can easily come across as public condemnation, and that is not a good thing.

As Felix said, at the very least, the result, and some rationale must be posted.

A few thoughts on what might help the public parts of the process:
- do not put the person's name/information in the title: that would at least stop it from being a "headline", so a title might be "Steering Committee member audit" or even "removal" instead of "audit" but without naming anyone
- do not focus the issue on a single individual at all (even if there is only one under discussion), and instead the entire issue from title to content is all about "evaluating inactive members", within the issue a list of members under evaluation is given. It's a small thing, but it's again about lessening the focus on pointing out a person
- the focus of what's posted publicly can be about the person's contributions and the time they dedicated; it can read more like a "farewell and thank you" with "we hope you'll return if time permits" etc. More of a celebration of what they have achieved? 
- all public comments (evidence of inactivity, etc.) could go through private discussion before being posted, several reasons for this:
  - it lets some of the discussion remain private with the explicit goal that it will be posted publicly in some form
  - it allows for more members to catch inflammatory or insensitive language (which will certainly be unintentional on the part of the author, it's a safety net)
  - many of our members, while not always obvious because their written English is fluent/near-native, are not native English speakers, and that can both make catching the above more difficult in their own writing, or make it easier to catch in others' because of differing perspective, so I think this committee wordsmithing makes whatever we post publicly stronger for everyone involved
  - we have at least one member whose main focus is documentation content and use of language, so we'll be better positioned all around to have publicly posted parts of this process protect the individual
  - I realize this may make the process slow though, if we're doing this comment by comment. I'm not necessarily suggesting it goes full vote for every comment. Or perhaps practically speaking, it means doing a single public posting at the end with a carefully worded summary/conclusion.

Reviewer 4: We should probably adopt a process somewhere between the "all is public from the start" and "we will do the work behind the scenes."

The initial step should probably be sending an email to all steering committee members stating the reasons for the decision. This email will give us the last opportunity to reach out to the unresponsive member and give them a chance to react.

I hope that once we get to that stage, the unresponsive member will either start contributing again or step down on their own. In this case, there is no need to open a ticket, and we can all move on with our lives one way or another.

Suppose the unresponsive member does not respond to an email. In that case, we should open an issue in the community topics repository, state the reasons there, and then vote just like we would on any other topic. This way, we retain transparency and keep permanent proof of the process. (I am assuming that GitHub will not go bust in the next few years.)

At least this is the best compromise I can come up with.

Author: @felixfontein @briantist @tadeboro thanks for your thoughts! I've elaborated on the process and have changed it in accordance with your ideas. PTAL

@samccann @acozine you're opinions here will be much appreciated, could you please take a look too?
